### DISCUSSION TITLE
Possible to change the Service and Pod CIDR ?

### DISCUSSION THREAD

**Discussion by jackserippi** [kubesphere/kubesphere#6150] [Q&A]
I wounder if it is Possible to change the CIDR of the Service and Pods. If yes can anybody help me to do so?

**Comment by calvinyv** [comment]
you could try:
–service-cluster-ip-range string
A CIDR notation IP range from which to assign service cluster IPs. This must not overlap with any IP ranges assigned to nodes or pods. Max of two dual-stack CIDRs is allowed.
